Added version notes for Vim/Neovim and AppImage fuse solution (#38)
nstankov-bg/docs.plutus-community.com: book/docs/setup/editors/vim.md

sudo apt install vim
```
For the other platforms check: https://www.vim.org/

> Make sure that you have version >=8.1.1719 or coc.nvim plugin will not work properly.

Reviewer: ```suggestion
Make sure that you have version >=8.1.1719 or the `coc.nvim` plugin will not work properly.
```

```bash
nvim
```

> If you get error related to `libfuse` or `fusemount` you should install fuse with:

Reviewer: ```suggestion
If you get an error related to `libfuse` or `fusemount`, you should install fuse with:
```

```bash
sudo apt install fuse
```

> On the other Linux distribution then Debian-like can be package name `fuse-utils`. 

Reviewer: ```suggestion
On other Linux distributions than Debian (or alike), the package name could be `fuse-utils`. 
```
